server_admin/topics/authentication/x509.adoc

```diff
@@ -209,6 +209,61 @@ image:images/x509-directgrant-flow.png[]
 
 image:images/x509-directgrant-flow-bindings.png[]
 
+==== Client certificate lookup
+
+When user sends HTTP requests directly to {project_name} server, it's the {appserver_name} undertow subsystem, which will establish the SSL handshake and extracts client certificate. Client certificate will be then saved to the attribute `javax.servlet.request.X509Certificate` of the HTTP request as specified in the servlet specification. {project_name} X509 authenticator will be then able to lookup the certificate from this attribute.
```

Comment:
When an HTTP request is sent directly to {project_name} server, the {appserver_name} undertow subsystem will establish an SSL handshake and extract the client certificate. The client certificate will be then saved to the attribute `javax.servlet.request.X509Certificate` of the HTTP request, as specified in the servlet specification. The {project_name} X509 authenticator will be then able to lookup the certificate from this attribute.